src/components/table/table-selection.ts

```diff
@@ -170,7 +179,7 @@ export class TableSelection {
         }
     };
 
-    private getActiveRows: () => Tabulator.RowComponent[] = () => {
+    private getActiveRows: () => TabulatorRowComponent[] = () => {
```

Comment:
https://github.com/user-attachments/assets/124bd8f0-fd18-48f0-93af-ea13cc3ad7f9

in the web client, checking the boxes will also open the sidepanel. They are considered to be a click on the row

Reply:
Fixed in f52d3d474abd5a93d3bac4889040dfb92dbcde9b